src/Connections/GithubRepo.ts

```diff
@@ -489,9 +509,7 @@ export class GitHubRepoConnection extends CommandConnection<GitHubRepoConnection
                 "github",
             );
         this.hookFilter = new HookFilter(
-            AllowHookByDefault,
-            state.enableHooks,
-            state.ignoreHooks,
+            state.enableHooks ?? DefaultHooks,
```

Comment:
We need this, otherwise a lack of defaults on missing state would be a breaking change.

Reply:
Added stronger typing.